### PR TITLE
Improve NIC name detection

### PR DESCRIPTION
**Pull request by sardaukar**

### CODE REVIEW

**Comment by sardaukar** [comment]
Not sure how to test this...

**Comment by woudsma** [comment]
So:
```sh
ETH_LINK=$(ip -o route show to default | awk '{print $5}')
NODE_IP=$(ifconfig $ETH_LINK | grep -oP 'inet \K\S+')

echo $NODE_IP
```
prints your server IP address?

**Comment by sardaukar** [comment]
@woudsma it does on Vultr. Pretty sure on DO too

**Comment by woudsma** [comment]
@sardaukar Yes it does :)

First contribution 🎉 Thanks a lot!

**Comment by sardaukar** [comment]
I should try to improve the NODE_IP one, too. Using ifconfig is deprecated :| gimme a few mins?

File: install [hunk 1]
```diff
@@ -23,12 +23,12 @@ FPREFIX="=====>"
 PREFIX="----->"
 INDENT="      "
 
-REPOSITORY="https://github.com/swarmlet/swarmlet.git"
+REPOSITORY="https://github.com/sardaukar/swarmlet.git"
```
Comment (woudsma) [inline install:26]:
```suggestion
REPOSITORY="https://github.com/swarmlet/swarmlet.git"
```

**Comment by woudsma** [comment]
@sardaukar Does `ifconfig` still work in your setup though? I wasn't aware it was deprecated.

**Pull request merged**